chia/full_node/full_node.py

```diff
@@ -394,7 +394,7 @@ def set_server(self, server: ChiaServer):
             self.log.error(f"Exception in peer discovery: {e}")
             self.log.error(f"Exception Stack: {error_stack}")
 
-    def _state_changed(self, change: str, change_data: Dict[str, Any] = None):
+    def _state_changed(self, change: str, change_data: Dict[str, Any] = {}) -> None:
```

Comment:
Odds are decent that it is ok to have the same dict reused on each call, but I think it might be better to avoid the nuance and the risk of this changing in the future.  The rest of the path from here seems to be using `Optional` as well so it can just be passed through still.

```suggestion
    def _state_changed(self, change: str, change_data: Optional[Dict[str, Any]] = None) -> None:
```

Reply:
good point, fixed